PowerShell/ScubaGear/Modules/Providers/ExportSharePointProvider.psm1

```diff
@@ -10,10 +10,12 @@ function Export-SharePointProvider {
     param (
         [Parameter(Mandatory = $true)]
         [ValidateSet("commercial", "gcc", "gcchigh", "dod", IgnoreCase = $false)]
+        [ValidateNotNullOrEmpty()]
         [string]
         $M365Environment,
 
         [Parameter(Mandatory = $false)]
+        [ValidateNotNullOrEmpty()]
```

Comment:
In gneral a type Switch does nto need to be validated.  It does not acceept input - it either exists or not.